utils/node_local_test.py

```diff
@@ -3883,7 +3886,7 @@ def _prep(self):
 
             if not finished:
                 while len(active) < max_child:
-                    active.append(self._run_cmd(fid))
+                    active.append(self._run_cmd(fid, valgrind=True))
```

Comment:
This line needs reverting before landing, with this in place all fault injection tests are run under valgrind which affects the runtime.